### PR TITLE
Adding purple berets for R&D

### PR DESCRIPTION
**Pull request by lzk228**
:cl:
- add: Added purple berets for R&D!
- Screenshot_20230316_094543
Screenshot_20230316_094557
Screenshot_20230316_094606

### CODE REVIEW

**Comment by DogZeroX** [comment]
Make sure the add: says "add: Added purple berets for science!" or something like that.

**Comment by Flareguy** [comment]
Why is the beret a different color from the regular science purple?

**Comment by potato1234x** [comment]
imo this would be better in rd's locker

**Comment by deltanedas** [comment]
yeah to go in line with ce/cmo/hos/warden(?) berets

File: Resources/Textures/Clothing/Head/Hats/beret_rnd.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "Taken from tgstation at commit https://github.com/tgstation/tgstation/commit/4f6190e2895e09116663ef282d3ce1d8b35c032e",
```
Comment (AJCM-git) [inline Resources/Textures/Clothing/Head/Hats/beret_rnd.rsi/meta.json:4]: you need to specify whatever changes you made to the sprite here

**Review by AJCM-git — CHANGES_REQUESTED** [review]

**Comment by lzk228** [comment]
I'd rather make a new PR

**Pull request closed without merging**